server/src/main/java/org/opensearch/action/admin/cluster/remotestore/stats/TransportRemoteStoreStatsAction.java

```diff
@@ -49,6 +49,8 @@ public class TransportRemoteStoreStatsAction extends TransportBroadcastByNodeAct
     RemoteStoreStats> {
 
     private final IndicesService indicesService;
+
+    private final ClusterService clusterService;
```

Comment:
this is not being used, lets remove this.

Reply:
Ack